builds/misc/packages-release.yaml

```diff
@@ -21,6 +21,10 @@ stages:
             arch: amd64
             os: centos7
             target.iotedged: edgelet/target/rpmbuild/RPMS/x86_64
+          RedHat8-amd64:
```

Comment:
You should also add this to Publish Packages Github Stage

Reply:
+1